scss/mixins/_list-group.scss

```diff
@@ -2,21 +2,31 @@
 
 // List Groups
 
+// Boosted mod: only add an icon on the right instead of changing the cackground-color
 // scss-docs-start list-group-mixin
-@mixin list-group-item-variant($state, $background, $color) {
+@mixin list-group-item-variant($state, $color, $icon) {
```

Comment:
Is it me or is this mixin modified here but never actually used? And shouldn't be removed from the migration guide as a deprecation warning?

Reply:
Yeah the mixin isn't used anymore but I felt like if people were using it anyway, it'd be better to convey the same logic as the list-group variant.

Reply:
It's not possible when the interface/parameters changes, so let's let it as it's in _main_ branch